editor/src/clj/editor/changes_view.clj

```diff
@@ -179,7 +179,7 @@
       (when (some? git)
         (.close git)))))
 
-(defn make-changes-view [view-graph workspace prefs async-reload! ^Parent parent]
+(defn make-changes-view [view-graph workspace prefs ^Parent parent async-reload!]
```

Comment:
Changed argument order to `make-changes-view`, putting the `async-reload!` callback in the last argument position.